core/src/main/java/jenkins/tools/GlobalToolConfiguration.java

```diff
@@ -24,6 +24,7 @@
 package jenkins.tools;
 
 import com.google.common.base.Predicate;
+import hudson.BulkChange;
```

Comment:
BulkChange doesn't appear to be used here?

Reply:
indeed, fixed